i18n/zh-CN/docusaurus-plugin-content-docs/current/install/cluster-deployment/k8s-deploy/compute-storage-coupled/install-config-cluster.md

```diff
@@ -768,3 +768,22 @@ Doris 集群在部署后设置了 root 用户的密码，需要配置一个拥
 - 部署后设置 root 密码，并配置新的拥有管理节点的用户名和密码后，会引起存量服务滚动重启一次。    
 :::
 
+
+:::tip 提示
+- 部署后设置 root 密码，并配置新的拥有管理节点的用户名和密码后，会引起存量服务滚动重启一次。    
+  :::
```

Comment:
This format is incorrect. All the document formats below will be in the prompt box